pandas/core/window/ewf.py

```diff
@@ -59,11 +59,11 @@ def get_center_of_mass(
     return float(comass)
 
 
-class EWM(_Rolling):
+class Exponential_moving(_Rolling):
```

Comment:
could you use `ExpontentialMoving` other than `Expontential_moving`?

Reply:
sure